server/database/build.sql

```diff
@@ -0,0 +1,23 @@
+BEGIN;
+DROP TABLE IF EXISTS users, tickets_status, tickets CASCADE;
+CREATE TABLE users(
+    id SERIAL PRIMARY KEY,
+    email TEXT NOT NULL,
+    name VARCHAR NOT NULL UNIQUE,
```

Comment:
the unique value should be the email, not the name. What if two people are named mohammed?

Reply:
Ohhhh , ok 
But I wrote unique in name instead email .
thank you for this comment